bids/layout/tests/test_path_building.py

```diff
@@ -66,16 +65,25 @@ def test_insufficient_entities(layout, strict, validate):
         ("qmri_mtsat"),
         ("qmri_sa2rage"),
         ("qmri_vfa"),
-        ("ds000117"),
+        ("synthetic"),
     ],
 )
-def test_path_building_on_examples_with_derivatives(dataset, scope, bids_examples):
+def test_path_building_on_derivative_examples(dataset, bids_examples):
     layout = BIDSLayout(bids_examples / dataset, derivatives=True)
-    files = layout.get(subject=".*", datatype=".*", regex_search = True, scope=scope)
-    for bf in files:
-        entities = bf.get_entities()
-        path = layout.build_path(entities)
-        assert(path==bf.path)
+    for derivative in layout.derivatives:
+        sublayout = layout.derivatives[derivative]
+        files = sublayout.get(subject=".*", datatype=".*", regex_search=True)
+        for bf in files:
+            entities = bf.get_entities()
+
+            # Some examples include unfinalized derivatives
+            if {"label"} & set(entities):  # BEP011
+                continue
```

Comment:
So that would skip the mask and segmentation as they are currently in the spec, correct?

Do you want me to add them to the config ?

Reply:
If you're up for it, sure!